MEP_Engine/Compute/Error.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+using BH.oM.Base;
+using BH.oM.Reflection.Attributes;
+using BH.oM.MEP.Fixtures;
+using BH.oM.Architecture.Elements;
+using BH.Engine.Reflection;
+
+namespace BH.Engine.MEP
+{ 
+    public static partial class Compute
+    {
+        /******************************************/
+        /****         Record an error          ****/
+        /******************************************/
+
+        internal static void NonPolylineElementError()
+        {
+            Reflection.Compute.RecordWarning("The method works only with Polyline, since it needs to a closed perimeter.");
+        }
+
+        /******************************************/
+        
+        internal static void NonClosedElementError()
+        {
+            Reflection.Compute.RecordWarning("The method works only with closed curves Polyline.");
+        }
```

Comment:
I don't like this kind of approach to this, and this isn't even being used anywhere in the rest of the code. Please delete it.